flake.nix

```diff
@@ -44,7 +44,7 @@
             cronosd-testnet = mkApp packages.cronosd-testnet;
             stateless-testcase = {
               type = "app";
-              program = "${pkgs.testground-testcase}/bin/stateless-testcase";
+              program = "${pkgs.stateless-testcase}/bin/stateless-testcase";
```

Comment:
I think the `stateless` alone is strange, `testground` or just `benchmark` makes more sense?